Documentation/ProjectFile/prj/processes/process/RICHARDS_FLOW/t_mass_lumping.md

```diff
@@ -1 +1 @@
-\ogs_missing_documentation
+A tag for using mass-lumping scheme
```

Comment:
I'm not familiar with mass-lumping scheme. Is this tag for enabling/disabling mass-lumping scheme or is it used for choosing a specific mass-lumping scheme? It would be nice if the documentation is more meaningful.